amm/repl/src/test/scala/ammonite/session/EvaluatorTests.scala

```diff
@@ -380,5 +380,31 @@ object EvaluatorTests extends TestSuite{
         z2: Z[X] = Z(X(7))
       """)
     }
+    test("extension-methods"){
```

Comment:
I'm wondering `EvaluatorTests` is the right place to test this.

Reply:
`AdvancedTests` should work, in particular it has similar tests (like `"given"` which ensures users can declare givens in Scala 3).

Reply:
Sounds better, moved this test to `AdvancedTests` 👍